integration-tests/testkit/seed.ts

```diff
@@ -210,9 +210,12 @@ export function initSeed() {
                 ownerToken,
               ).then(r => r.expectNoGraphQLErrors());
 
-              const targets = projectResult.createProject.ok!.createdTargets;
-              const target = targets[0];
               const project = projectResult.createProject.ok!.createdProject;
+              const targets = projectResult.createProject.ok!.createdTargets.map(_ => ({
```

Comment:
nit: `_` typically represents an unused argument. Prefer `target`

Reply:
Would `$` work for you? Or do you prefer something explicit every single time?